subsys/bluetooth/host/crypto.c

```diff
@@ -111,6 +136,102 @@ int bt_rand(void *buf, size_t len)
 }
 #endif /* CONFIG_BT_HOST_CRYPTO_PRNG */
 
+#if defined(CONFIG_BT_USE_PSA_API)
```

Comment:
can you make a new file `crypto_psa.c` ? You can rename that one to `crypto_tc.c`